File: sample/src/main/java/com/example/java/IfElseDenesting.java
package com.example.java;

public class IfElseDenesting {
    public void foo() {
        if (true) {
            return;
        }
        System.out.println();
        System.out.println();
    }

    public void bar() {
        if (true) {
            System.out.println();
        } else if (false) {
            if (false) {
                System.out.println();
            }
        } else {
            if (true) {
                System.out.println();
            }
        }
    }
}

File: sample/python/if_else_denesting.py
def foo(condition: bool):
    asd = 0 == 1
    if condition:
        print()
    else:
        print()
        print()


def bar(condition: bool):
    if condition:
        print(
            0,
        )

def baz(condition: bool):
    if condition:
        if condition:
            print()
            return
        else:
            print()
